test/testcafe/spec/ResetAuthenticatorPasswordView_spec.js

```diff
@@ -68,7 +68,7 @@ test
     await t.expect(resetPasswordPage.hasPasswordError()).eql(false);
     await t.expect(resetPasswordPage.getConfirmPasswordError()).eql('New passwords must match');
 
-    await t.expect(await resetPasswordPage.signoutLinkExists()).notOk();
+    await t.expect(await resetPasswordPage.signoutLinkExists()).ok();
```

Comment:
Another occurrence, not sure if this was intended..

@vincentvoong-okta - So this is the screen in SSPR flow where we prompt the user to enter the the password twice with the rules displayed. With this change, we will have a Back to sign in link in the footer, is this ok?